contrib/puppetlabs-lumogon/lib/puppet/provider/lumogon/lumogon.rb

```diff
@@ -0,0 +1,37 @@
+require 'json'
+
+Puppet::Type.type(:lumogon).provide(:lumogon) do
+  desc "Provider for Lumogon"
+  commands :docker => '/usr/bin/docker'
```

Comment:
docker isn't always at that path, for instance it's `/usr/local/bin/docker` when using Docker for X. I'd tried this locally with:

```ruby
commands :docker => 'docker'
```